process/css-styleguide/README.md

```diff
@@ -0,0 +1,168 @@
+
+```html
+<button class="btn btn-dodger-blue btn-secondary" type="button">Button</button>
+```
+
+4. To change the size of the button, you can add `btn-sm` or `btn-lg` to the class list.
+
+```html
+<button class="btn btn-sm btn-dodger-blue" type="button">Button</button>
+```
+
+
+Output :
+
+<img width="50%" alt="Capture d’écran 2022-08-23 à 10 33 08" src="https://user-images.githubusercontent.com/48251481/186461703-49f5d19b-128e-4b28-9348-f1838c69ab74.png">
+
+
+### Input
+
+1.  The parent class `.input` will assign some basic styling to the input.
+2.  Labels can also be styled depending upon various needs. The classes are `.input--label`, `.input--message`, `.input--error`, and `.input--message-error`
+
+```html
+<input
+  type="text"
+  class="input input--label"
+  placeholder="Write something"
+/>
+```
+
+Output :
+
+<img width="311" alt="Capture d’écran 2022-08-23 à 10 34 05" src="https://user-images.githubusercontent.com/48251481/186462336-24f8c55e-29fc-49c8-93c4-e45a90eb1e72.png">
+
+### Modal
+
+1. To render the content in a modal window, wrap it inside the parent element such as `div` and give it the `.modal` class.
+
+```html
+<div class="modal">
+    <div class="title title-l">This is a modal</div>
+</div>
+```
+
+Output :
+
+![Screenshot from 2022-07-11 15-08-37](https://user-images.githubusercontent.com/64376712/178236255-d999514e-50f9-4960-9958-497ce207d1ed.png)
+
+### Search form
+
+The search form is composed of 3 elements, each with its own corresponding classes:
+1. A form with a `.search-form` class, which gives flex container properties to the form element
+
+2. An input: the base class `.input` assign basic input style to the input. It needs to be combined with `search-input` to give it a search bar style, adding the magnifier glass to it.
+
+3. A reset button: the base class for styling a reset button is `.search-input-reset`.
+
+```html
+  <form class="search-form">
+    <input
+      type="search"
+      placeholder="Search something"
+      class="input search-input"
+    />
+    <button type="reset" class="search-input-reset">
+      <!-- Icon here  -->
+    </button>
+  </form>
+```
+
+Output :
+
+<img width="488" alt="Capture d’écran 2022-08-25 à 18 07 31" src="https://user-images.githubusercontent.com/48251481/186716079-47c20b88-f806-4daf-ac2a-95795067ed3e.png">
+
+### Tag
+
+1. Choose the styling of the tag depending upon the requirements by using different classes.
+   Classes list:
+   1. `.tag` (required - base class name)
+   2. `.tag-success`
+   3. `.tag-process`
+   4. `.tag-loader`
+   5. `.tag-error`
+   6. `.tag-delete`
+
+  👉 `.tag-loader` is the loader animation and is to be used with `.tag-process`
+
+```html
+<span class="tag tag-success body body-s">active</span>
+```
+
+Output :
+
+<img width="125" alt="Capture d’écran 2022-08-23 à 10 35 58" src="https://user-images.githubusercontent.com/48251481/186462461-fe47b488-be82-4b6b-8ef0-0469c5bb0c6f.png">
+
```

Comment:
Does the `creating` tag have the  `.tag-process` classname? If so, can we name it `process` in order to have consistency between the class names and the result we see in this image? 
Same for the `deleted` / `delete` tag :)

Can we also reorder them in the image, to make the reading easier? (same order between the list and the tags on the image):
<img width="750" alt="Capture d’écran 2022-08-29 à 11 46 35" src="https://user-images.githubusercontent.com/30866152/187174280-f98792d9-2e47-47f6-b8c9-ce8cdca2466a.png">